contracts/proposal/dao-proposal-single/src/contract.rs

```diff
@@ -273,18 +384,43 @@ pub fn execute_execute(
             &config.dao,
             Some(prop.start_height),
         )?;
-        if power.is_zero() {
+        let vetoer_call = match config.timelock {
```

Comment:
```suggestion
        let vetoer_call = match prop.timelock {
```

use `timelock` from proposal instead of config

Reply:
Very good catch.

Reply:
Very good catch.